Policies/github-terms-of-service.md

```diff
@@ -1504,7 +1504,10 @@ This is irrespective of:
  - whether notice is given.
      
 #### 2. Survival
-&dagger; All provisions of this Agreement which, by their nature, should survive termination *will* survive termination<BR>&nbsp;&nbsp;&nbsp;&nbsp;<sup>*(including [without limitation] ownership provisions, warranty disclaimers, indemnity, and limitations of liability).*</sup>
+&dagger; All provisions of this Agreement which, by their nature, should survive termination *will* survive termination<BR><TT><sup>&nbsp;&nbsp;&nbsp;&nbsp;*(including [without limitation]:&nbsp;&nbsp;•&nbsp;ownership&nbsp;provisions,</sup></TT><BR> 
+<TT><sup>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;•&nbsp;warranty&nbsp;disclaimers,</sup></TT><BR>   
```

Comment:
WUT?

Reply:
Hello @KOLANICH,

You can reject any of my suggestions. I'm not pinning my hopes on GitHub accepting any of my suggestions in particular. Also, my suggestions are part of a process of researching possible ways to rewrite contracts for making them more user friendly; in line with this, it's perfectly acceptable to reject my suggestions.

Having a look at what you have highlighted, I haven't as such&mdash;as far as I can tell&mdash;modified the sentence. Instead, I've tried to modify the previous visual formatting of the sentence, by use of indentation and carriage returns (a bit like code pretty printers), to try to make it easier to comprehend. Upon closer inspection, it appears you've highlighted an out-of-date modification (where I used incorrect encoding). The new version should be as displayed at the bottom of this comment.

Once again, you can reject any of my suggestions.
<HR>

```#### 2. Survival
&dagger; All provisions of this Agreement which, by their nature, should survive termination *will* survive termination<BR>
<sup>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;*(including [without limitation]:</sup><BR>
<sup>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;•&nbsp;ownership&nbsp;provisions,</sup><BR>
<sup>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;•&nbsp;warranty&nbsp;disclaimers,</sup><BR>
<sup>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;•&nbsp;indemnity,&nbsp;&nbsp;&nbsp;&nbsp;AND</sup><BR>
<sup>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;•&nbsp;limitations of liability).*</sup>
```

#### 2. Survival
&dagger; All provisions of this Agreement which, by their nature, should survive termination *will* survive termination<BR><sup>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;*(including [without limitation]:</sup><BR><sup>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;•&nbsp;ownership&nbsp;provisions,</sup><BR><sup>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;•&nbsp;warranty&nbsp;disclaimers,</sup><BR><sup>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;•&nbsp;indemnity,&nbsp;&nbsp;&nbsp;&nbsp;AND</sup><BR><sup>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;•&nbsp;limitations of liability).*</sup>